packages/main/src/TabInOverflow.hbs

```diff
@@ -4,6 +4,9 @@
 	type="{{this.overflowState}}"
 	?selected="{{this.effectiveSelected}}"
 	?disabled="{{this.effectiveDisabled}}"
+	?aria-disabled="{{this.effectiveDisabled}}"
+	aria-selected="{{this.effectiveSelected}}"
+	aria-labelledby="{{this.ariaLabelledBy}}"

```

Comment:
I think that aria-disabled should be either "true|false" or not set at all. In this case, with "?", it will remain just aria-disabled. 

There is also small difference in the VoiceOver announcement. Without the "?", when aria-disabled="true", VoiceOver mentions "dimmed" in addition to the item's text.

But, I am not an expert, you can double check it with our ACC experts.

Reply:
Yes, it is not correct to have only aria-disabled with no value. The 'dimmed' announcement is expected.